woocommerce/woocommerce-gutenberg-products-block
# Update review block icons

### Screenshots

![image](https://user-images.githubusercontent.com/3616980/63761105-db4c5680-c8c0-11e9-94ea-b04d17110f75.png)

### How to test the changes in this Pull Request:

1. Create a post and, when adding blocks, verify all review blocks have the correct icon.
